submit/controllers/tests/test_authorship.py

```diff
@@ -89,15 +126,16 @@ def test_post_request_with_data(self, mock_load, mock_save, mock_url_for):
         mock_url_for.return_value = redirect_url
 
         form_data = MultiDict({'authorship': 'y', 'action': 'next'})
-        data, code, headers = authorship('POST', form_data, submission_id)
+        data, code, headers = authorship('POST', form_data, self.session,
+                                         submission_id)
         self.assertEqual(code, status.HTTP_303_SEE_OTHER,
                          "Returns 303 redirect")
         self.assertEqual(headers['Location'], redirect_url,
                          "Location for redirect is set")
 
     @mock.patch('submit.controllers.util.url_for')
-    @mock.patch('events.save')
-    @mock.patch('events.load')
+    @mock.patch('arxiv.submission.save')
+    @mock.patch('arxiv.submission.load')
     def test_verify_fails(self, mock_load, mock_save, mock_url_for):
         """Event store flakes out on authorship verification."""
```

Comment:
Does that mean the data isn't good or that the connection isn't good? Or either? Or other cases?